specificatie/datum.yaml

```diff
@@ -69,12 +77,16 @@ components:
           properties:
             jaar:
               $ref: '#/components/schemas/Jaar'
+            langFormaat:
+              type: string
```

Comment:
Hier kun je ook gewoon verwijzen naar 

`$ref: '#/components/schemas/Jaar'`

Reply:
ik denk het niet. Jaar is een integer, dit moet een string zijn.

Reply:
Dat kan ik me van de andere locaties waar langFormaat voorkomt voorstellen maar waarom moet het hier persé een string zijn?

Als het toch string moet zijn dan kun je hier wel het pattern

^([1-2]{1})([0-9]{3})$

toevoegen.